### ISSUE TITLE
arista_eos_show_ip_route parse exception

### ISSUE THREAD

**Issue by centove** [networktocode/ntc-templates#811]
- Bug Report

##### TEMPLATE USING
```
Value Filldown VRF (\S+)
Value Filldown PROTOCOL (\S+\s\S+?|\w?)
Value Filldown NETWORK (\d+.\d+.\d+.\d+)
Value Filldown MASK (\d+)
Value Filldown DISTANCE (\d+)
Value Filldown METRIC (\d+)
Value DIRECT (directly)
Value Required NEXT_HOP (connected|\d+\.\d+\.\d+\.\d+)
Value INTERFACE (\S+)

Start
  ^\s+${PROTOCOL}\s+${NETWORK}/${MASK}\s+(?:\[${DISTANCE}/${METRIC}\]|is\s+${DIRECT})(?:.+?)${NEXT_HOP},\s+${INTERFACE}$$ -> Record
  ^\s+via\s+${NEXT_HOP},\s+${INTERFACE} -> Record
  ^VRF\s+name:\s+${VRF}\s*$$
  ^VRF:\s+${VRF}\s*$$
  ^WARNING
  ^kernel
  ^Codes:
  # Match for codes
  ^\s+\S+\s+-\s+\S+
  ^Gateway\s+of\s+last
  ^\s*$$
  ^. -> Error

```

##### SAMPLE COMMAND OUTPUT
```
VRF: default
Codes: C - connected, S - static, K - kernel, 
       O - OSPF, IA - OSPF inter area, E1 - OSPF external type 1,
       E2 - OSPF external type 2, N1 - OSPF NSSA external type 1,
       N2 - OSPF NSSA external type2, B - BGP, B I - iBGP, B E - eBGP,
       R - RIP, I L1 - IS-IS level 1, I L2 - IS-IS level 2,
       O3 - OSPFv3, A B - BGP Aggregate, A O - OSPF Summary,
       NG - Nexthop Group Static Route, V - VXLAN Control Service,
       DH - DHCP client installed default route, M - Martian,
       DP - Dynamic Policy Route, L - VRF Leaked

 C        162.220.49.0/26 is directly connected, Vlan105
 C        162.220.49.64/26 is directly connected, Vlan104
 C        162.220.49.128/31 is directly connected, Tunnel11, Static Interface GRE tunnel index 11, dst 184.170.244.12, src 216.201.90.212
 B I      162.220.49.130/31 [200/0] via 199.229.255.1, Vlan50
 B I      162.220.49.132/31 [200/0] via 199.229.255.1, Vlan50
 C        162.220.49.134/31 is directly connected, Tunnel12, Static Interface GRE tunnel index 12, dst 104.200.133.181, src 216.201.90.212
 S        162.220.49.0/24 is directly connected, Null0



```

##### SUMMARY
```
Get an exception trying to parse the above with the arista_show_ip_route template
textfsm.parser.TextFSMError: State Error raised. Rule Line: 23. Input Line:  C        162.220.49.128/31 is directly connected, Tunnel11, Static Interface GRE tunnel index 11, dst 184.170.244.12, src 216.201.90.212
##### STEPS TO REPRODUCE

show IP route <prefix> longer-prefixes and have a gre tunnel route in the output

```

##### EXPECTED RESULTS
Parsed output.

##### ACTUAL RESULTS
```
textfsm.parser.TextFSMError: State Error raised. Rule Line: 23. Input Line:  C        162.220.49.128/31 is directly connected, Tunnel11, Static Interface GRE tunnel index 11, dst 184.170.244.12, src 216.201.90.212
```

**Comment by itdependsnetworks** [comment]
I have updated the comment to better format the output.

**Comment by diepes** [comment]
PR #843  -  nexthop ip/int changed to List,  not parsing details in tunnel nexthop, just extracting it.

**Comment by FragmentedPacket** [comment]
Fixed via #843, not confirmed by user.
